src/main/java/core/basesyntax/Machine.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Machine {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present

Reply:
Hi , if i delete line 2 Maven  test failed